contents/tutorials/django-analytics.md

```diff
@@ -0,0 +1,416 @@
+
+from .views import PostListView
+
+urlpatterns = [
+    path('', PostListView.as_view(), name='home'),
+]
+```
+
+Finally, include the `blog` app URLs in the tutorial project `urls.py` file.
+
+```python
+# tutorials/urls.py
+
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('', include('blog.urls')),
+    path('admin/', admin.site.urls),
+]
+```
+
+Once you save all of this, reload your server, and open your [development site](http://127.0.0.1:8000/), you should see a page with a list of blog posts.
+
+![Blog](../images/tutorials/django-analytics/blog.png)
+
+## 3. Adding PostHog
+
+With a basic Django app all setup, we can add PostHog to track user behavior. We can do this in two ways, adding the snippet and using the Python library.
+
+### Adding the snippet
+
+PostHog makes it easy to get started fast with autocapture. This captures events like pageviews, button and link clicks, field inputs, allows for session recordings, and more. 
+
+> To enable session recordings, toggle on “Record user sessions”  in project settings under “Recordings”
+
+In a new project’s getting started flow or your project settings, copy the HTML snippet and add it into the `<head>` tag of your `base.html` page. 
+
+![Snippet](../images/tutorials/django-analytics/snippet.png)
+
+```html
+<!DOCTYPE html>
+<head>
+    <title>My cool blog</title>
+    <script>
+        !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
+        posthog.init('<ph_project_api_key>',{api_host:'<ph_instance_address>'})
+    </script>
+</head>
+<html>
+    <body>
+        <h1>My cool blog</h1>
+        {% for post in posts %}
+            <h2>{{ post.title }}</h2>
+            <p>Created: {{ post.created_at }}</p>
+            <p>Author: {{ post.author }}</p>
+            <p>{{ post.content }}</p>
+        {% endfor %}
+        Logged in as: <b>{{user.email}}</b>
+    </body>
+</html>
+```
+
+Once you save and reload, autocapture immediately starts working. Refreshing the page gives you a `pageview` event, and you’ll be all setup to capture more events as you add more functionality.
```

Comment:
Maybe add a point here about how to confirm it works by visiting the live events tab

```suggestion
Once you save and reload, autocapture immediately starts working. Refreshing the page gives you a `pageview` event, and you’ll be all setup to capture more events as you add more functionality. To verify that your `pageview` events are captured correctly, login to PostHog and check the Live Events feed via the sidebar. 
```